server/azure-local-service/README.md

```diff
@@ -0,0 +1,24 @@
+# azure-local-service
+
+Azure Local service is a minimal, self-contained, test implementation of the Azure Fluid Relay service that can be run locally and used for development/testing Fluid functionality in conjunction with the `TinyliciousClient` and `AzureClient` in local mode.
```

Comment:
While this statement is true we should make our messaging strong that `TinyliciousClient -> tinylicious` `AzureClient -> azure-local-service` to avoid confusion.

```suggestion
Azure Local service is a minimal, self-contained, test implementation of the Azure Fluid Relay service that can be run locally and used for development/testing Fluid functionality in conjunction with the `AzureClient` in local mode.
```